libs/schemas/src/events/events.schemas.ts

```diff
@@ -387,6 +387,16 @@ export const events = {
     }),
   }),
 
+  NamespaceTransferSingle: ChainEventBase.extend({
```

Comment:
It doesn't seem like you use a majority of the data provided as part of the `ChainEventBase` event so maybe it's best to not extend ChainEventBase and just pass the parameters  you need.